src/Auth/SignUp.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+
+import { AuthForm } from "./Auth.interface";
+import Input from "src/components/Input";
+import * as routes from "src/common/constants/pageRoutes";
+import Button from "src/components/Button";
+import { SignUpContainer } from "./Auth.styles";
+
+const SignUp: React.FunctionComponent<RouteComponentProps<any>> = ({
```

Comment:
Are we going to integrate these pages with the Auth Module in a separate PR?

Reply:
Yes this is Auth-UI only